ts/output/common/Wrappers/mglyph.ts

```diff
@@ -70,6 +77,11 @@ export function CommonMglyphMixin<T extends WrapperConstructor>(Base: T): Mglyph
      */
     public valign: number;
 
+    /**
+     * TextNode used for deprecated fontfamily/index use case
```

Comment:
That should be an `@override`

Reply:
Is that because of the interface?  If so, shouldn't the other ones above also be `@override`?

Reply:
And the `getParameters()` below?

Reply:
Yes. `@override` ensures that the documentation is inherited from the definition in the superclass/interface. 
I had not spotted the `getParameters`, as the interface method already existed (and I clearly missed it the first time around). Effectively the `@override` can replace the first line of the comment in the method implementation.

Reply:
OK.  I'm afraid there are lots of files that have this wrong, then.  It would be good to do a documentation sweep before the next major release.